ui/Makefile

```diff
@@ -192,7 +192,7 @@ audit: package.json package-lock.json  ## Audit package dependencies for vulnera
 .PHONY: test
 test:  ## Run tests
 	@echo "${INFO}Running tests"
-	CI=true $(NPM) test -- --env=jsdom --coverage --silent
+	CI=true $(NPM) test -- --env=jsdom --coverage --maxWorkers=100% --silent
```

Comment:
`maxWorkers=100%` seems to speed up tests on average by 1 minute but some people in forums have complained that this can cause instability in tests, such as tests timing out or not finishing before going onto the next test, however in my testing this did not occur at all. Thoughts?

Reply:
IMHO, 1 minute is unlikely worth the risk of the complication, especially if it's possible that the test would fail silently and move on to the next test. In that case, there is the risk of a developer not noticing that tests aren't functioning properly.

Reply:
Removed! @breedenc